schemas/com.apple.swiftui/window_group/jsonschema/1-0-0

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "com.apple.swiftui",
+    "name": "window_group",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for a window group entity, representing the SwiftUI window group that the event occurs in.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
```

Comment:
I am wondering since this ID will come from the user (it's not generated by the tracker), whether we should be so strict and require it to be in the UUID format? Might be easier for users in case they already have some other identifier of the window, to just pass that. What do you think?

Reply:
I think it's good to have the UUID option, but it would probably work better to make the string ID the required one yes (since string ID is used to make the WindowGroup)

Reply:
Should we name it something like `windowId` (instead of `uuid`) to make it clear what it identifies?